src/Frontend.hs

```diff
@@ -69,6 +70,7 @@ aulaTop (Nat runAction) =
   :<|> (\req cont -> getSamplesPath >>= \path ->
           waiServeDirectory path req cont)
   :<|> waiServeDirectory (Config.config ^. htmlStatic)
+  :<|> redirect "/space"
```

Comment:
I disagree.  If you try to hit a page and don't have access, you should be redirected to `/login`.  If you are loading `/`, you should be pointed to some page that is most likely to be usedful directly **after** login.  Even better than `/space` would be the current year's class idea space for students, the admin workbench for admins, and so on.  but this is still sci-fi...